src/block-management/ethrpc-transport-adapter.js

```diff
@@ -18,6 +18,9 @@ function createTransportAdapter(transporter) {
       getBlockByHash: function (hash, callback) {
         dispatch(eth.getBlockByHash([hash, false], errorSplittingWrapper(callback)));
       },
+      getBlockByNumber: function (n, callback) {
```

Comment:
Suggest using a more descriptive parameter name like `blockNumber`, instead of `n`.

Reply:
Good call